benchmarks/test_power_nu.py

```diff
@@ -4,6 +4,10 @@
 
 import pyccl as ccl
 
+# NOTE: We now check up to kmax=23
+# because CLASS v3 has a slight mismatch
+# (6e-3) at higher k wavenumbers.
+KMAX = 23
```

Comment:
We install `class < 3` though.

Reply:
This was failing on my own computer, and I imagine it might for some users.

Reply:
That's because you used `class > 3` I assume. Are tests and benchmarks supposed to be run by users too?
If yes, should we perhaps create a `test_cll.sh` at the root directory which automatically runs benchmarks and tests?

Reply:
Users don't have to run them, but they often do, and this precludes unnecessary issues being opened. I don't think the script is needed.